integreat_cms/cms/forms/pages/page_translation_form.py

```diff
@@ -23,3 +27,26 @@ class Meta:
         model = PageTranslation
         #: The fields of the model which should be handled by this form
         fields = MachineTranslationForm.Meta.fields + ["slug", "hix_score"]
+
+    def clean_slug(self):
+        """
+        Validate the slug field (see :ref:`overriding-modelform-clean-method`)
+
+        :return: A validated slug
+        :rtype: str
+        """
+        cleaned_slug = self.cleaned_data["slug"]
```

Comment:
```suggestion
        super().clean_slug()
        cleaned_slug = self.cleaned_data["slug"]
```
This overwrites a the `clean_slug` method at https://github.com/digitalfabrik/integreat-cms/blob/2c560214ffb3a8af52435dec32192d7dca730a21/integreat_cms/cms/forms/custom_content_model_form.py#L170-L181